llvm/test/Transforms/LoopVectorize/vplan-noinbounds-gep.ll

```diff
@@ -0,0 +1,107 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 3
+; RUN: opt -passes=loop-vectorize -force-vector-width=4 -enable-vplan-native-path -S %s | FileCheck %s
```

Comment:
```suggestion
; RUN: opt -passes=loop-vectorize -force-vector-width=4 -S %s | FileCheck %s
```

no nested loops, so no the 'native' path isn't used.

Reply:
done